exonum-java-binding/pom.xml

```diff
@@ -426,7 +426,29 @@
           <configuration>
             <source>${java.compiler.source}</source>
             <doclint>none</doclint>
+            <!-- Allow fetching links below -->
+            <isOffline>false</isOffline>
             <detectOfflineLinks>false</detectOfflineLinks>
+            <links>
+              <!-- Published Exonum API docs. Only the modules that are dependencies of
+                   other modules are listed here.
+
+                   To generate release Javadocs, a two-step process is required:
```

Comment:
Should we add this to release template in wiki?

Reply:
Yes, it makes sense.